### ISSUE TITLE
Value indicator triggers re-rendering of datasets

### ISSUE THREAD

**Issue by ennerf** [GSI-CS-CO/chart-fx#476]
**Describe the bug**
Moving a value indicator triggers a layout of the entire chart. This is a big performance hit for large datasets.

**To Reproduce**
Run `ChartIndicatorSample` and move the indicator
 
**More Information**
The problem seems to be that the marker gets added to the plot foreground (see [AbstractSingleValueIndicator](https://github.com/GSI-CS-CO/chart-fx/blob/cd202c7c9a0b40f1e66b6d0fe8fd93fffbec5435/chartfx-chart/src/main/java/de/gsi/chart/plugins/AbstractSingleValueIndicator.java#L239-L253)), which triggers a layout of the chart area.

Removing the marker makes it work as expected.

```java
var indicator = new XValueIndicator(chart.getXAxis(), 0, "") {
    @Override
    protected void layoutMarker(double startX, double startY, double endX, double endY) {
        // don't add a marker to avoid re-rendering the chart area
    }
};
chart.getPlugins().add(indicator);
```

I'm not familiar enough with the scene graph layout. Is it possible to have it in the axis area without causing re-rendering? If not, do you think it would make sense to have a flag for hiding the marker for the few cases where it matters?

**Comment by wirew0rm** [comment]
Hey, thanks for bringing this up, this was changed in 11.2.0 [here](https://github.com/GSI-CS-CO/chart-fx/commit/9f4a563ca77a09989668194ea7ce508930a0842c#diff-b981098b2e3524f8edd4c170b07cafcf85645ad8ad2d4112f025e14195c2a401L237-R250) because it was necessary to see to which axis an indicator belonged to when there were multiple axes. I guess there should be a way to make the node not trigger the relayout, but i'll have to investigate how to do that.

**Comment by wirew0rm** [comment]
solved in #485
